index.php

```diff
@@ -0,0 +1,51 @@
+<?php
+?>
+
+<!DOCTYPE HTML>
+<HTML>
+    <head>
+        <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+        <link rel="stylesheet" type="text/css" href="styles/styles.css" >
+    </head>
+    <body>
+        <h1 id="page title">Your Dream Diary</h1>
+        <h2 id="dreams title">Dreams you have had</h2>
+        <h3 id="dream title">Walls in house made of sponge cake!</h3>
+        <h4 id="dream or nightmare">Dream/Nightmare</h4>
+        <h4>01/01/1970</h4>
+        <p id="dream description">I had a dream where I saw cracks appearing in the walls of my house, 
+            that started to grow and grow. As they grew, they started to look like sponge cake,
+             and fell away to reveal huge holes in the wall. How odd!! I spoke with my therapist about this,
+             and she said it might be a metaphore about all of the things that I am learning, and feeling like 
+             my brain almost doesn't have space to store it all! I'm not toally sure I agree with this, but who 
+             knows, she is the pro....
+        </p>
```

Comment:
not sure, but I think all the whitespace in the <p> tags, which I assume you put in for the indentation in the code,  could lead to unforeseen problems down the road?

Reply:
sorry:
not sure, but I think all the whitespace in the p
tags, which I assume you put in for the indentation in the code, could lead to unforeseen problems down the road?

Reply:
done!